public/app/features/alerting/unified/components/rules/RulesFilter.tsx

```diff
@@ -104,18 +112,17 @@ const RulesFilter = ({ onFilterCleared = () => undefined }: RulesFilerProps) =>
   const handleAlertStateChange = (value: PromAlertingRuleState) => {
     logInfo(LogMessages.clickingAlertStateFilters);
     updateFilters({ ...filterState, ruleState: value });
-  };
-
-  const handleViewChange = (view: string) => {
-    setQueryParams({ view });
+    trackRulesSearchComponentInteraction('ruleState');
```

Comment:
Q: Can we track all filter updates in `updateFilters()` instead of adding it to each individual interaction?

Reply:
Resolved offline :)